### ISSUE TITLE
Add a Role="aplication" somewhere in the Client

### ISSUE THREAD

**Issue by frastlin** [mozilla/hubs#2703]
**Is your feature request related to a problem? Please describe.**
When a screen reader user opens the client, they just see the buttons and edit box. There is no option that says "Application" to use the wasd keys to move.

**Describe the solution you'd like**
1. Please add a `role="application"` to a div with a button or some kind of focusable element that can receive input. That way, when we hit enter or tab into the application, we can passs all keys into the app.
2. Have commands, such as "/" open up a chat box and have escape or enter close the chat box.
3. Have shortcut keys to do all the actions with an aria live region announcing actions.
4. If the user turns on chat announcements, have each new chat read as an aria live region.
5. Have a key command, such as "," and "." read past messages in an aria live region.

**Describe alternatives you've considered**
The current method works for users who know there should be WASD key commands to move, and who are programmers who understand that not all buttons will pass their keystrokes to the parent element. But for most users, especially new users, they will have no idea how to get the WASD keys to work.

**Comment by emclaren** [comment]
I wonder if this is related to this issue: https://github.com/mozilla/hubs/issues/2205

**Comment by frastlin** [comment]
Yes, this solves #2205 and should put Jaws into forms mode and NVDA into input mode. VO on OSX will work fine on this as well, but VO on IOS has no way of passing gestures to the application yet.

**Comment by frastlin** [comment]
Take note that once you enter an application element, then the only aria that's useful is aria live regions. So a custom interface to access the room's features will need to be built utilizing aria live regions, key commands, and the web speech API.
This is because ARIA has no VR component type yet.

**Comment by TechThomas** [comment]
Please know that myself and my team at https://meetup.com/a11yvr will be happy to help test this once it is implemented
